regions/core/core.py

```diff
@@ -161,6 +161,36 @@ def to_shapely(self):
         """
         raise NotImplementedError("")
 
+    @abc.abstractmethod
+    def make_patch(self, **kwargs):
```

Comment:
I would call this ``as_mpl_patch`` (we've used this name in another package, but I can't remember which one now!)